public/app/features/dashboard/components/AnnotationSettings/AnnotationSettingsEdit.tsx

```diff
@@ -116,6 +184,26 @@ export const AnnotationSettingsEdit = ({ editIdx, dashboard }: Props) => {
             <ColorValueEditor value={annotation?.iconColor} onChange={onColorChange} />
           </HorizontalGroup>
         </Field>
+        <Field label="Show in" aria-label={selectors.pages.Dashboard.Settings.Annotations.NewAnnotation.showInLabel}>
+          <>
+            <Select options={panelFilters} value={panelFilter} onChange={onFilterTypeChange} inputId={FILTER_TYPE_ID} />
+            {panelFilter !== PanelFilterType.AllPanels && (
+              <MultiSelect
+                options={panels}
+                value={panels.filter((panel) => annotation.filter?.ids.includes(panel.value!))}
+                onChange={onAddFilterPanelID}
+                isClearable={true}
+                placeholder="Choose panels"
+                width={100}
+                closeMenuOnSelect={false}
+                className={styles.select}
+                inputId={CHOOSE_PANELS_ID}
+              />
+            )}
+          </>
+        </Field>
+      </FieldSet>
+      <FieldSet>
```

Comment:
Not necessary for this PR, but in future we could refactor this out into a separate component for improved readability / maintainability